### ISSUE TITLE
Too short segment not displayed in table of viewpoint tab

### ISSUE THREAD

**Issue by hansenp** [TheJacksonLaboratory/Gopher#201]
If I run Gopher with the simple approach, I find examples in which the upstream or downstream segments are not adjacent to the center segment:

![image](https://user-images.githubusercontent.com/10495485/43151455-3dd4a65c-8f6c-11e8-8680-31457b96082f.png)

In this case the adjacent upstream segment is too short and cannot be selected. It should appear in the table with a disabled checkbox but selected segment (green) should not appear in the table, because only adjacent segments can be added to the center segment by definition.

**Comment by hansenp** [comment]
We decided to keep unselectable digests in the list of segments of given viewpoints. Therefore, we changed in class ViewPoints this
```
            if(!restFrag.isUnselectable()) {
                restrictionSegmentList.add(restFrag);
            }
```
to this

`restrictionSegmentList.add(restFrag);`

But now it is again as before (with an if statement). If I remove the if statement everything works as expected.

@pnrobinson: Is this intended or did this happen accidentally?

**Comment by pnrobinson** [comment]
I agree with removing it -- I think the "if" was just experimental and it got forgotten.
